src/plurals.js

```diff
@@ -26,3 +26,14 @@ export const calculateDelimiters = delimiter => ({strings, msgstr, names, values
 
   return {lengthUntranslated, lengthTranslated, groups};
 };
+
+
+export const defaultSplitPlural = msgid => msgid.split('|');
+defaultSplitPlural.expect = 2;
```

Comment:
Validate the correct number of delimiters. Don't want `splitPlural` to return Error since that wont work for production. Could move it to another `option` if that is less weird.